scripts/install

```diff
@@ -85,7 +22,7 @@ ynh_add_nginx_config
 #=================================================
 # INSTALL COUCHDB
 #=================================================
-ynh_script_progression --message="Installing CouchDB..." --weight=60
+ynh_script_progression --message="Installing $app..." --weight=60
 
 COUCHDB_PASSWORD=$password
 echo "couchdb couchdb/mode select standalone
```

Comment:
Hmf so right now the tests are failing because we need to declare those debconf stuff ... I guess we gonna have to try to download the .deb manually and install it with `dpkg` >_>